libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -129,6 +129,7 @@ class AP_ExternalAHRS {
         uint8_t instance;
         float pressure_pa;
         float temperature;
+        float altitude;
```

Comment:
What frame is this in?  Above geoid?  AMSL?

Comment if nothing else

Reply:
Done, I added a comment